apps/docs/app/(docs)/[...slug]/page.tsx

```diff
@@ -23,6 +23,13 @@ export async function generateMetadata({
 	return metadata
 }
 
+export async function generateStaticParams() {
+	const paths = await db.getAllPaths()
+	return paths
+		.filter((path) => !path.startsWith('/blog') && !path.startsWith('/legal/'))
```

Comment:
i'm worried we'll forget about this in the future if there's another top-level folder. possible to add something that checks against an array of paths it should ignore?